packages/rx_bloc_cli/lib/src/commands/create_command.dart

```diff
@@ -214,6 +214,18 @@ class CreateCommand extends Command<int> {
 
     _progressFinish(format, progress);
 
+    progress = _logger.progress(
+      'dart fix --apply .',
```

Comment:
let's communicate this through the change log.

Reply:
Done